src/components/Form/utils/validators.ts

```diff
@@ -0,0 +1,22 @@
+import { FormRule as AntFormRule } from 'antd'
+
+export type FormRule = AntFormRule
+
+export const required = (message?: string): FormRule => {
+  return {
+    message: message || 'The field is required',
+    required: true,
+  }
+}
+
+export const checkEquals = (fieldName: string, message?: string): FormRule => {
```

Comment:
I would put a JSDoc over this, so that users of the library know how to use it well

Reply:
sure! we are going to do this in the actual implementatation step

Reply:
added jsdoc in https://github.com/mia-platform/design-system/blob/IDP-94/form-analysis/src/components/Form/Validators/index.ts